ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -609,6 +649,8 @@ DTC:
   is_abstract: true
   implements:
   - OPERATIONAL
+  opt_uses:
+  - discharge_air_damper_percentage_command
```

Comment:
This doesn't feel relevant to this abstract type. Please explain.

Reply:
discharge_air_damper_percentage_command removed from DTC. Added discharge_air_damper_percentage_command in AHU_DTC_CHWVM_H3X_OAMC_DFVSC_UV_CO2C2X_ZHM_ZTM and AHU_DFVSC_CHWVM_CLPM_DTC_UV_CO2C_MTM_OAFM_RHM_WDT